src/controllers/dynakube/activegate/generate.go

```diff
@@ -0,0 +1,23 @@
+package activegate
+
+import "github.com/pkg/errors"
+
+const (
+	CommunicationEndpointsName = "communication-endpoints"
+	TenantTokenName            = "tenant-token"
+	TenantUuidName             = "tenant-uuid"
+)
+
+func (r *Reconciler) GenerateData() (map[string][]byte, error) {
```

Comment:
Most methods and functions in any programming language generate data. Can you rename it to reflect its purpose more precisely?